proposed/2021/NuGet-org-login-auth-for-dotnet-nuget-push.md

```diff
@@ -0,0 +1,79 @@
+# NuGet.org login authentication workflow for `dotnet nuget push`
+
+- Author: Christopher Gill (https://github.com/chgill-MSFT)
+- Start Date: 2021-03-1
+- GitHub Issue: https://github.com/NuGet/Home/issues/10657
+- GitHub PR: N/A
+
+## Summary
+
+Currently there are 3 ways to push NuGet packages to NuGet.org:
+* [Manual upload on NuGet.org](https://www.nuget.org/packages/manage/upload)
+* `donet nuget push` with a valid API key
+* `nuget push` with a valid API key
+
+I propose creating a workflow to push to NuGet.org via the `dotnet nuget push` command by NuGet.org account login in the CLI or the browser.
+
+## Motivation 
+
+1. API keys can be a pain to use in many circumstances, especially for beginners. To securely upload a package to NuGet.org via the CLI you need to:
```

Comment:
Do you think it would be useful to run usability studies on our current `nuget push` flow to validate this hypothesis?